sql-cli/tests/test_opensearch_connection.py

```diff
@@ -39,6 +39,7 @@ def test_query(self, connection):
         )
 
     @estest
+    @pytest.mark.skip(reason="Test is not compatible with OpenSearch >= 2.3.0, it returns HTTP/503 instead of HTTP/400")
```

Comment:
I've left a comment on the bug to revert this if it's fixed: https://github.com/opensearch-project/sql/issues/851